--- a/include/freetype/config/ftoption.h	2018-09-17 17:21:31.577941426 +0200
+++ b/include/freetype/config/ftoption.h	2018-09-17 17:21:23.891274464 +0200
@@ -658,7 +658,7 @@
   /* [1] https://www.microsoft.com/typography/cleartype/truetypecleartype.aspx */
   /*                                                                       */
 /* #define TT_CONFIG_OPTION_SUBPIXEL_HINTING  1         */
-#define TT_CONFIG_OPTION_SUBPIXEL_HINTING  2
+/* #define TT_CONFIG_OPTION_SUBPIXEL_HINTING  2         */
 /* #define TT_CONFIG_OPTION_SUBPIXEL_HINTING  ( 1 | 2 ) */
 
 
